components/raftstore/src/store/fsm/store.rs

```diff
@@ -180,6 +184,10 @@ pub struct StoreMeta {
     pub region_read_progress: RegionReadProgressRegistry,
     /// record sst_file_name -> (sst_smallest_key, sst_largest_key)
     pub damaged_ranges: HashMap<String, (Vec<u8>, Vec<u8>)>,
+    /// record peers in recovery progress
+    pub pending_recovery_peers: HashSet<u64>,
```

Comment:
I think `recovery` is a bit ambiguous. It could be recovery from a snapshot or recovery from corrupted data.

Reply:
How about `pending apply`

Reply:
Falsely closed it, accepted. Thx.